care/emr/resources/report/template/spec.py

```
@@ -0,0 +1,108 @@
+        return v
+
+    @field_validator("context")
+    @classmethod
+    def validate_context(cls, v):
+        if not v:
+            msg = "Report Type is required"
+            raise ValueError(msg)
+        try:
+            DataPointRegistry.get(v)
+        except KeyError as e:
+            raise ValueError("Invalid Context type") from e
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)